src/index.ts

```diff
@@ -5,6 +5,9 @@ const extension = new NetlifyExtension();
 
 extension.addEdgeFunctions("./src/edge-functions", {
   prefix: "sa_plugin",
+  shouldInjectFunction: () => {
+    return process.env["ENABLE_SIMPLE_ANALYTICS"] === "true";
+  },
```

Comment:
We don't want to use the `ENABLE_SIMPLE_ANALYTICS` all together right?

Reply:
the said the following:

> H: Default of a setting that affects site build or behavior must be “off”

this is the only way to satisfy that. 

this probably because Netlify itself (still) lacks fine grained controls when installing extensions. e.g. GitHub, Vercel, let you either install for the entire team or select projects

Reply:
Yes, but this is the whole extension. Just remove it please. I see already issues from customers that tell us: it does not work! And then it's that missing env param all the time.